test/tests/functional/pbs_soft_walltime.py

```diff
@@ -860,8 +858,9 @@ def test_soft_held(self):
         self.server.rerunjob(jid)
         self.server.expect(JOB, {'job_state': 'H'}, id=jid)
 
-        # sleep so long to verify that soft_walltime don't change
-        # while job is held
+        self.logger.info(
+            "sleep so long to verify that soft_walltime" +
+            " don't change while job is held")
```

Comment:
I think you don't need the '+'.  Python will take two quoted strings between () and combine them together.  I believe you do need the '+' when you don't have the ().